docs/site/decorators/Decorators_openapi.md

```diff
@@ -626,6 +626,44 @@ class MyOtherController {
 }
 ```
 
+### @oas.visibility
+
+[API document](https://loopback.io/doc/en/lb4/apidocs.openapi-v3.visibility.html)
+
+This decorator can be applied to class and/or a class method. It will set the
+`x-visibility` property, which dictates if a class method appears in the OAS3
+spec. When applied to a class, it will mark all operation methods of that class,
+unless a method overloads with `@oas.visibility(<different visibility type>)`.
+
+Currently, the supported values are `documented` or `undocumented`.
```

Comment:
Please mention what is the default value used when `@oas.visibility` was not called. I think we should explain that by default, `x-visibility` metadata is not added (the value is `undefined`), which is interpreted by LB as `visibility: documented`.